README.md

```diff
@@ -42,13 +42,14 @@ Note that all third-party extensions are removed upon upgrading.
 
 ## Adding extensions after installation
 
+You need before to install [Composer](https://getcomposer.org/download/).
 Replace `flarum` with your app ID in case of multiple installation.
 Replace `vendor/extension` with the appropriate names. Read the extension documentation if it requires additional steps.
 
 ```bash
 app=flarum
 cd /var/www/$app
-sudo -u $app php7.3 composer.phar require vendor/extension
+sudo php composer.phar require -n vendor/extension
```

Comment:
I do not really agree here either. The default `php` version could be something different than `php7.3`, so we need to call it explicitely.

Regarding adding `-n`, since this action is manual, maybe it is wise to let the user be prompted any necessary question?